src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
I don't think the DefaultHtmlGenerator case is the same as this. In that one there seems to be a real functional difference between the two, and we really don't want people calling the old ctor anymore. The same is true for OutputFormatterCanWriteContext - the old ctor won't really work.

But in this case here both ctors work great, they have no functional difference.

As to the null refs, maybe I'm missing it, but the property could always have been null already anyway, no? If so, there's no change in behavior there either...